include/deal.II/grid/tria_accessor.h

```diff
@@ -2489,6 +2496,116 @@ class CellAccessor :  public TriaAccessor<dim,dim,spacedim>
   /**
    * @}
    */
+  /**
+   * @name Dealing with periodic neighbor
+   */
+  /**
+   * @{
+   */
+  /**
+   * If the cell has a periodic neighbor at its @c ith face, this function
+   * returns true, otherwise, the returned value is false.
+   */
+  bool has_periodic_neighbor(const unsigned int i) const;
+
+  /**
+   * For a cell with its @c ith face at a periodic boundary,
+   * (see @ref GlossPeriodicConstraints "the entry for periodic baoundaries")
+   * this function returns an iterator to the cell on the other side
+   * of the periodic boundary. If there is no periodic boundary at the @c ith
+   * face, an exception will be thrown.
+   * In order to check if a cell has periodic neighbor on its @c ith face,
+   * one should first call the function @c has_periodic_neighbor().
```

Comment:
Maybe reword: In order to avoid running into an exception, check the result of has_periodic_neighbor() for the ith face prior to this function.